parts/windows/windowscontainerdfunc.ps1

```diff
@@ -160,17 +160,19 @@ function Install-Containerd {
     $zipfile = [Io.path]::Combine($ENV:TEMP, "containerd.zip")
     DownloadFileOverHttp -Url $ContainerdUrl -DestinationPath $zipfile
     Expand-Archive -path $zipfile -DestinationPath $global:ContainerdInstallLocation -Force
-    del $zipfile
+    Remove-Item -Path $zipfile -Force
+
   }
   elseif ($ContainerdUrl.endswith(".tar.gz")) {
     # upstream containerd package is a tar 
     $tarfile = [Io.path]::Combine($ENV:TEMP, "containerd.tar.gz")
     DownloadFileOverHttp -Url $ContainerdUrl -DestinationPath $tarfile
     Create-Directory -FullPath $global:ContainerdInstallLocation -DirectoryUsage "storing containerd"
     tar -xzf $tarfile -C $global:ContainerdInstallLocation

```

Comment:
I think that we can remove elseif. tar -xzf does not work in Windows node.

Reply:
tar -xzf should work in Windows node. We used this command the uncompress the `https://acs-mirror.azureedge.net/containerd/windows/v0.0.1/binaries/containerd-v0.0.1-windows-amd64.tar.gz`

Reply:
Also, this works on my Windows PC

Reply:
It seems like that tar.exe is installed by default in Windows.